practice-one/src/Components/Section/Navbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from '../Common/Buttons';
+
+const Navbar = () => (
+  <div className=" bg-light">
+    <nav className="container mx-auto w-screen h-[80px]  ">
+      <div className="px-2 flex justify-between items-center w-full h-full">
+        <div className="flex items-center gap-5">
+          <a className="navbar-brand logo" href="#">
+            <img src="src\assets\logo.png" alt="logo" />
+          </a>
+          <ul className="hidden ps-20 gap-10  md:flex">
+            <li><a href="/Product" className="text-base hover:text-primary">Product</a></li>
+            <li><a href="/Contact" className="text-base hover:text-primary">Pricing</a></li>

```

Comment:
Check the href link, It's should be #contact

Reply:
Checked and fixed